api/admin/package.json

```diff
@@ -8,11 +8,7 @@
   },
   "author": "NYPL",
   "license": "Apache-2.0",
-  "engines": {
-    "node": ">=8.0.0",
-    "npm": ">=5.0.0"
-  },
   "dependencies": {
-    "simplified-circulation-web": "0.5.16"
+    "simplified-circulation-web": "0.6.0-rc3"
```

Comment:
@keithbauer one more update! Can you set this to `0.6.0`?
I'll leave it as that for now. It reduced issues flagged by dependabot from 53 to 30 but it's good enough for now...
I want to reassess the spreadsheet and make another release later on but this version and the update you have in this PR moves us along quite far already.

Reply:
Sounds good, if I can get some of these python dependabot updates working I'll add them in a separate PR.